meteor/client/main.html

```diff
@@ -9,7 +9,7 @@
 	<link rel="apple-touch-icon" sizes="180x180" href="/icons/apple-touch-icon.png" />
 	<link rel="icon" type="image/png" sizes="32x32" href="/icons/favicon-32x32.png" />
 	<link rel="icon" type="image/png" sizes="16x16" href="/icons/favicon-16x16.png" />
-	<link rel="manifest" href="/site.webmanifest" data-href="/site.webmanifest" />
+	<link rel="manifest" href="/site.webmanifest" data-href="./site.webmanifest" />
```

Comment:
This doesn't work properly.
If you load the page `https://my-domain/sofie/settings/tools/snapshots`, the client tries to find this at `https://my-domain/sofie/settings/tools/site.webmanifest?lng=en-GB`, which is not correct.